homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -130,6 +130,11 @@ def update(self):
             if equaliser >= len(temescal.equalisers):
                 temescal.equalisers.append("unknown " + str(equaliser))
 
+        # Temporary fix until handling of unknown function settings is integrated in the temescal library
+        for function in self._functions:
+            if function >= len(temescal.functions):
+                temescal.functions.append("unknown " + str(function))
+
```

Comment:
There is nothing more permanent than a temporary fix.

We should not add this to Home Assistant, instead the upstream library needs to be fixed first.

Reply:
Ok, i will make a pull request for the original library. Would it be ok to keep this pull request (and later merge it) in case the library author does not respond in the next couple of weeks?